docs/content/administration/reverse-proxies.en-us.md

```diff
@@ -39,21 +59,13 @@ server {
 }
 ```
 
-### Resolving Error: 413 Request Entity Too Large
-
-This error indicates nginx is configured to restrict the file upload size,
-it affects attachment uploading, form posting, package uploading and LFS pushing, etc.
-You can fine tune the `client_max_body_size` option according to [nginx document](http://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size).
-
```

Comment:
There might be some public links targeting this anchor. We sure we want to remove it?

Reply:
I do not think it's worth to add a lot of links into our document. We shouldn't document every 3rdparty option.

Reply:
> There might be some public links targeting this anchor.

I am not sure what it means. If you mean the "header" in our document, I do not think it's worth to keep it, why users need it? Without it, does it cause any confusion?

Reply:
I mean the header anchor. It could be that other docs link to it, so removing it may result in a partially dead link because the anchor is removed. Not a big issue but I thought to mention id.

Reply:
I do not think we ever guaranteed that the anchors are always there. No document site has such guarantee.

Reply:
Maybe move this block into the faq?

Reply:
1. It is already in FAQ, it just duplicates
2. I still keep the content, see the `## Nginx` section.

I do not see any problem here.

<details>

![image](https://github.com/go-gitea/gitea/assets/2114189/c36ca692-018a-42f7-a294-9f6ea54556d2)

</details>